src/Features/CSharp/Portable/ConvertToRecord/CSharpConvertToRecordRefactoringProvider.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.CodeAnalysis.CSharp.ConvertToRecord
     [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.ConvertToRecord), Shared]
     internal sealed class CSharpConvertToRecordRefactoringProvider : CodeRefactoringProvider
```

Comment:
Is it possible to derive from `SyntaxEditorBasedCodeFixProvider`?

Reply:
@Youssef1313 Added to https://github.com/dotnet/roslyn/issues/63849 to track this.